documentation/en/cli-lotus-miner.md

```diff
@@ -92,26 +99,27 @@ USAGE:
    lotus-miner run [command options] [arguments...]
 
 OPTIONS:
-   --miner-api value     2345
-   --enable-gpu-proving  enable use of GPU for mining operations (default: true)
-   --nosync              don't check full-node sync status (default: false)
-   --manage-fdlimit      manage open file limit (default: true)
-   --help, -h            show help
+   --miner-api                    2345 (default: "")
+   --enable-gpu-proving           enable use of GPU for mining operations (default: "true")
+   --nosync                       don't check full-node sync status (default: "false")
+   --manage-fdlimit               manage open file limit (default: "true")
+   --help, -h                     show help (default: "false")
+
 ```
 
 ## lotus-miner stop
+
 ```
 NAME:
    lotus-miner stop - Stop a running lotus miner
 
 USAGE:
    lotus-miner stop [command options] [arguments...]
 
-OPTIONS:
```

Comment:
7. probably should add an implicit `--help` to everything.

Reply:
Done.